CHANGES

```diff
@@ -9,6 +9,10 @@
 
  Changes between 1.1.1 and 3.0.0 [xx XXX xxxx]
 
+  *) Added X509_to_X509_REQ_ex function. It is allows to copy extensions
```

Comment:
` It allows copy of extensions ..` will be better IMHO.

Reply:
fixed